From 55862e3026b519ec7cd094785c4706443d0a8395 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Romain=20Roff=C3=A9?= <rofferom@gmail.com>
Date: Fri, 18 Oct 2024 20:03:17 +0200
Subject: [PATCH] configure: Disable FORTIFY_SOURCE

---
 configure | 40 ----------------------------------------
 1 file changed, 40 deletions(-)

diff --git a/configure b/configure
index 4ca76b5..366c8cf 100755
--- a/configure
+++ b/configure
@@ -15757,44 +15757,6 @@ rm -f core conftest.err conftest.$ac_objext conftest.beam \
 
 fi
 
-if test x$ac_cv_c_compiler_gnu = xyes
-then :
-
-    { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking whether to add -D_FORTIFY_SOURCE=2 to CFLAGS" >&5
-printf %s "checking whether to add -D_FORTIFY_SOURCE=2 to CFLAGS... " >&6; }
-    cat confdefs.h - <<_ACEOF >conftest.$ac_ext
-/* end confdefs.h.  */
-
-
-                int main() {
-                #ifndef _FORTIFY_SOURCE
-                    return 0;
-                #else
-                    this_is_an_error;
-                #endif
-                }
-
-
-_ACEOF
-if ac_fn_c_try_link "$LINENO"
-then :
-
-            { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: yes" >&5
-printf "%s\n" "yes" >&6; }
-            CFLAGS="$CFLAGS -D_FORTIFY_SOURCE=2"
-
-else $as_nop
-
-            { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: no" >&5
-printf "%s\n" "no" >&6; }
-
-fi
-rm -f core conftest.err conftest.$ac_objext conftest.beam \
-    conftest$ac_exeext conftest.$ac_ext
-
-
-fi
-
 CFLAGS="$CFLAGS -W"
 
 warn_CFLAGS="-Wall -Wextra -Wcast-align -Wnested-externs -Wshadow -Wstrict-prototypes"
@@ -18316,5 +18278,3 @@ printf "%s\n" "$as_me:
  Type \"make; make install\" to compile and install
  Type \"make check\" to run the test suite
 " >&6;}
-
-
-- 
2.39.5

